cron/src/tests/conftest.py

```diff
@@ -0,0 +1,162 @@
+                'available_status': BookStatus.RESERVED
+            },
+            {
+                'id': 3,
+                'asset_code': '3',
+                'library_item_id': 1,
+                'shelf': 'None',
+                'has_cd_disk': False,
+                'available_status': BookStatus.RESERVED
+            },
+            {
+                'id': 4,
+                'asset_code': '4',
+                'library_item_id': 1,
+                'shelf': 'None',
+                'has_cd_disk': False,
+                'available_status': BookStatus.RESERVED
+            },
+            {
+                'id': 5,
+                'asset_code': '5',
+                'library_item_id': 2,
+                'shelf': 'None',
+                'has_cd_disk': False,
+                'available_status': BookStatus.BORROWED
+            },
+            {
+                'id': 6,
+                'asset_code': '6',
+                'library_item_id': 2,
+                'shelf': 'None',
+                'has_cd_disk': False,
+                'available_status': BookStatus.BORROWED
+            }
+        ])
+
+    connection.execute(
+        rental_log.insert(), [
+            {
+                'id': 1,
+                'copy_id': 2,
+                'users_id': 1,
+                'book_status': BookStatus.RESERVED,
+                '_reservation_end': datetime(2030, 5, 4),
+                '_return_time': None
+            },
+            {
+                'id': 2,
+                'copy_id': 3,
+                'users_id': 1,
+                'book_status': BookStatus.RESERVED,
+                '_reservation_end': datetime(2030, 5, 5),
+                '_return_time': None
+            },
+            {
+                'id': 3,
+                'copy_id': 4,
+                'users_id': 1,
+                'book_status': BookStatus.RESERVED,
+                '_reservation_end': datetime(2030, 5, 7),
+                '_return_time': None
+            },
+            {
+                'id': 4,
+                'copy_id': 1,
+                'users_id': 1,
+                'book_status': BookStatus.BORROWED,
+                '_reservation_end': datetime(2030, 5, 7),
+                '_return_time': datetime(2030, 5, 7)
+            },
+            {
+                'id': 5,
+                'copy_id': 5,
+                'users_id': 1,
+                'book_status': BookStatus.BORROWED,
+                '_reservation_end': datetime(2030, 5, 7),
+                '_return_time': datetime(2030, 5, 7)
+            },
+            {
+                'id': 6,
+                'copy_id': 6,
+                'users_id': 2,
+                'book_status': BookStatus.BORROWED,
+                '_reservation_end': datetime(2030, 5, 7),
+                '_return_time': datetime(2030, 5, 7)
+            }
+        ]
+    )
+
+
+@pytest.fixture()
+def data_access_layer():
+    dal = DataAccessLayer('sqlite:///:memory:')
```

Comment:
I think it might be better to use postgres in test, because we use it in production. It is up to you if you change it.